addons/main/functions/debug/fnc_debugDraw.sqf

```diff
@@ -156,9 +126,9 @@ private _fnc_DrawRect = {
              };
              private _lastSeen = _unit getVariable [QGVAR(debug_LastSeenPos), [0, 0, 0]];
              _targetKnowledge append [
-                "Target Knowledge:<br/>",
-                "    Last Seen: ", _lastSeen, " (", round ((_knowledge select 2) *100)/100, ")<br/>",
-                "    Position Error: ", round ((_knowledge select 5) *100)/100, "<br/>"
+                "<t color='#C7CCC1'>Target Knowledge: </t><br/>",
```

Comment:
one t tag should be enough to encapsulate this text. no need to open and close it everytime